exporters/scss/src/js/generators/typography.ts

```diff
@@ -32,15 +31,25 @@ export function generateTypography(
       }
     });
 
-    const subfamily = token.value.font.subfamily.toLowerCase();
     const fontSize = printUnit(Math.round((token.value.fontSize.measure / defaultFontSize) * 1000) / 1000, 'rem');
     let fontStyle = 'normal';
-    let fontWeight = 'normal';
-    if (subfamily === 'italic') {
+    let fontWeight = +token.value.font.subfamily;
+
+    // Font Ebony has a different font weight mapping, so we remap these values directly
```

Comment:
Please, explain to me, what is different and why the map cannot be used anymore.

Reply:
we no longer need to remap all font weights as they are already returned from Supernova in the numeric format. 

I updated the code so it uses a map and not two ifs.